src/components/pages/observation/list/views/stats/table.tsx

```diff
@@ -46,15 +41,15 @@ export default function LifeListTable({ data, title, loadMoreUniqueSpecies, filt
               <tr key={`${specieName}`} className="fade">
```

Comment:
`<tr key={`${specieName}`} className="fade">`
can you refactor this as well, since key doesn't need to be in template literals this can be just be key={speciesName}